bedrock/pocket/views.py

```diff
@@ -7,9 +7,9 @@
 
 def server_error_view(request, template_name="pocket/500.html"):
     """500 error handler that runs context processors."""
-    return l10n_utils.render(request, template_name, ftl_files=["500"], status=500)
+    return l10n_utils.render(request, template_name, ftl_files=["pocket/500"], status=500)
```

Comment:
**Question (non-blocking):** Should we also add these templates to the `pocket/urls.py` file?

Reply:
Based on the way Bedrock's 404/500 templates are handled, this is the only place they need to be set.